pkg/build/builder/sti.go

```diff
@@ -265,8 +267,8 @@ func (s *S2IBuilder) Build() error {
 		} else {
 			glog.V(2).Infof("No push secret provided")
 		}
-		glog.V(1).Infof("Pushing %s image ...", pushTag)
-		if err := pushImage(s.dockerClient, pushTag, pushAuthConfig); err != nil {
+		fmt.Fprintf(s.out, "Pushing %s image ...\n", pushTag)
```

Comment:
i don't understand that, our default log level is "2" i believe, you should see any glog.V(2) and below messages....  what messages *were* you seeing?